File: src/test/java/streams/SumNumbersTest.java
package streams;

import org.junit.jupiter.api.Test;

import java.util.List;

import static org.junit.jupiter.api.Assertions.*;

class SumNumbersTest {
    @Test
    void getTotal() {
        int total = SumNumbers.getTotal(
                List.of(3, 1, 4, 1, 5, 9, 2, 6, 5));
        assertEquals(2 * (3 + 9 + 6), total);
    }
}

File: src/test/kotlin/sequences/SumNumbersTest.kt
package sequences

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test
import org.junit.jupiter.api.assertAll

internal class SumNumbersTest {

    @Test
    fun `total should be 36`() {
        assertAll(
                { assertEquals(36, getTotal(listOf(3, 1, 4, 1, 5, 9, 2, 6, 5))) },
                { assertEquals(36, getTotalUsingSequences(listOf(3, 1, 4, 1, 5, 9, 2, 6, 5))) }
        )
    }
}